packages/edit-widgets/src/blocks/legacy-widget/index.php

```diff
@@ -9,12 +9,22 @@
  * Register legacy widget block.
  */
 function register_block_core_legacy_widget() {
-	register_block_type_from_metadata(
+	$block_type     = register_block_type_from_metadata(
```

Comment:
Does using block metadata give us anything at this point? Should we do away with registering the block using metadata altogether and use `register_block_type`? No need, I think, to pretend that this is a normal block with normal block conventions 🙂